hedera-node/hedera-file-service-impl/src/main/java/com/hedera/node/app/service/file/impl/handlers/FileGetContentsHandler.java

```diff
@@ -36,9 +53,16 @@
  */
 @Singleton
 public class FileGetContentsHandler extends PaidQueryHandler {
+    private final NetworkInfo networkInfo;
+
     @Inject
     public FileGetContentsHandler() {
-        // Exists for injection
+        // TODO: Not sure how to get the network info here.
```

Comment:
Is this something that the handle context needs to be provide? If so, it should be added to the Handle Design Notion doc

Reply:
can you add a link to the Notion page you are referring to?

Reply:
https://www.notion.so/swirldslabs/Handle-Design-89035f4f8df449b4b38839203507a373?pvs=4

Reply:
added there thank you